### PR TITLE
add Sebastian as release manager

### PR DESCRIPTION
**Pull request by migueldiascosta**

### CODE REVIEW

**Review by boegel — APPROVED** [review]
lgtm

I'll leave it to @SebastianAchilles to merge

**Review by SebastianAchilles — APPROVED** [review]
Thanks a lot for adding me as release manager @migueldiascosta 

lgtm

**Comment by SebastianAchilles** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**
